app/components-react/windows/go-live/DestinationSwitchers.tsx

```diff
@@ -80,17 +85,20 @@ function DestinationSwitcher(p: IDestinationSwitcherProps) {
   const containerRef = useRef<HTMLDivElement>(null);
   const platform = typeof p.destination === 'string' ? (p.destination as TPlatform) : null;
   const { RestreamService, MagicLinkService } = Services;
+  const canEnableRestream = RestreamService.views.canEnableRestream;
+  const canDisableDestination = canEnableRestream ? true : !p.isPrimary;
 
   function onClickHandler(ev: MouseEvent) {
-    if (p.isPrimary) {
+    if (!canDisableDestination) {
```

Comment:
if we're only checking negatives for booleans we should probably reformulate the boolean to be a positive identifier ie `cannotDisableDestination`